app/lib/frontend/handlers/package.dart

```diff
@@ -76,7 +79,10 @@ Future<shelf.Response> packageVersionsListHandler(
       final dartSdkVersion = await getDartSdkVersion();
       return renderPkgVersionsPage(
         data,
-        versions.versions,
+        // The cached version list contains the versions in semantically ascending order.
+        // By reversing that list, we will have them in semantically descending order,
+        // which is the preferred order for displaying on the versions page.
```

Comment:
This should also be documented in the documentation comment for `packageBackend.listVersionsCached`, especially if it promises to always return the versions in said order.

Reply:
It is documented on the `versions` field when we are accessing, I think that's a better place for it, but we can also repeat it at other places.